sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionTestEnvironment.cs

```diff
@@ -1,22 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
 using Azure.Core.TestFramework;
-using Azure.Identity;
 
 namespace Azure.Monitor.Ingestion.Tests
 {
     public class MonitorIngestionTestEnvironment : TestEnvironment
     {
-        public string DCRImmutableId => GetRecordedVariable("AZURE_MONITOR_INGESTION_LOGS_DCR_RULE_ID");
+        public string DCRImmutableId => GetRecordedVariable("INGESTION_DATA_COLLECTION_RULE_IMMUTABLE_ID");
 
-        public string DCREndpoint => GetRecordedVariable("AZURE_MONITOR_INGESTION_DATA_COLLECTION_ENDPOINT");
+        public string DCREndpoint => GetRecordedVariable("INGESTION_DATA_COLLECTION_ENDPOINT");
 
-        public string Ingestion_WorkspaceId => GetRecordedVariable("AZURE_MONITOR_INGESTION_INGESTION_WORKSPACE_ID");
+        public string Ingestion_WorkspaceId => GetRecordedVariable("WORKSPACE_ID");
```

Comment:
I think we probably want INGESTION in the name in case other services have a similar env var.

Reply:
both Monitor.Query and Monitor.Ingestion use the same value

Reply:
Ah ok